x-pack/legacy/plugins/reporting/server/routes/lib/reporting_feature_pre_routing.ts

```diff
@@ -20,7 +21,7 @@ export const reportingFeaturePreRoutingFactory = function reportingFeaturePreRou
   return function reportingFeaturePreRouting(getReportingFeatureId: GetReportingFeatureIdFn) {
     return function licensePreRouting(request: Legacy.Request) {
       const licenseCheckResults = xpackMainPlugin.info.feature(pluginId).getLicenseCheckResults();
-      const reportingFeatureId = getReportingFeatureId(request);
+      const reportingFeatureId = getReportingFeatureId(request) as string;
```

Comment:
NIT: the signature seems to already define the return type, why was the force-cast necessary?

`export type GetReportingFeatureIdFn = (request: Legacy.Request) => string;`